draft-ietf-quic-load-balancers.md

```diff
@@ -1119,6 +1131,14 @@ server ID allocations for the clear text and the encrypted versions.
 
 These attacks cannot be mounted against the Single Pass Encryption algorithm.
 
+## Early deletion of load balancer connection state
+
+Potential vulnerabilities related to heuristics that delete per-connection state
+are described in {{per-connection-state}}. Under certain assumptions about
+server configuration and fallback algorithm, this state might be critical to
+maintaining connectivity. Under other assumptions, the state provides robustness
+to improbable network events.
```

Comment:
I'm finding this text to be a little obtuse.  Maybe its jet lag speaking, but I'm not following.  Could you consider making this a little more direct?

Reply:
I guess I passed over that because Martin D. and I had extended discussions of the "advanced LB" algorithms and I recognized the issue. In the "no configuration" corner case, the LB learns the server CID by tracking packets sent by the client -- if the source CID matches the source CID of packets sent with destination CID = ICID, the destination CID has to be either the ICID or the server selected CID. But of course hackers can send packets that pretend to come for the client, and inject noise in that insecure algorithm.

I guess we should describe this type of attacks in the security section. The mitigation is to make sure that the server gets a proper configuration. I do not have a good mitigation otherwise.